python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -254,8 +262,7 @@ def to_json(self) -> Dict:
             "filter_expr": self.filter_expr,
         }
 
-    @staticmethod
-    def _validate(features: Sequence[Union[str, Feature]]) -> None:
+    def _validate(self, features: Sequence[Union[str, Feature]]) -> None:
```

Comment:
Should we check that pre-sliding features and sliding features only use fields from the source table, like what this PR validates in DerivedFeatureView?